src/process_forest.py

```diff
@@ -111,11 +121,43 @@ def get_process_from_4689_event(self):
         ppid = int(self.get_xpath("/Event/System/Execution").get("ProcessID"), 10)
         cmdline = "UNKNOWN"
         ppname = "UNKNOWN"
+        hashes = "UNKNOWN"
         user = self.get_xpath("/Event/EventData/Data[@Name='SubjectUserName']").text
         domain = self.get_xpath("/Event/EventData/Data[@Name='SubjectDomainName']").text
         logonid = self.get_xpath("/Event/EventData/Data[@Name='SubjectLogonId']").text
         computer = self.get_xpath("/Event/System/Computer").text
-        p = Process(pid, ppid, cmdline, ppname, path, user, domain, logonid, computer)
+        p = Process(pid, ppid, cmdline, ppname, hashes, path, user, domain, logonid, computer)
+        p.end = self._record.timestamp()
+        return p
+        
+    def get_process_from_1_event(self):
```

Comment:
this function name is a little confusing. makes me think it gets data from 1 event, rather than from event with eid 1.